<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>946</width>
    <height>892</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <widget class="QDoubleSpinBox" name="doubleSpinBox">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>40</y>
           <width>150</width>
           <height>20</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>20</height>
          </size>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>60</y>
           <width>150</width>
           <height>20</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>20</height>
          </size>
         </property>
        </widget>
        <widget class="QLabel" name="aLabel">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>190</y>
           <width>16</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>a</string>
         </property>
        </widget>
        <widget class="QLabel" name="bLabel">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>220</y>
           <width>16</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>b</string>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>40</y>
           <width>131</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Толщина мишени, см</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>60</y>
           <width>131</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Толщина защиты, см</string>
         </property>
        </widget>
        <widget class="QTextEdit" name="aTextEdit">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>190</y>
           <width>91</width>
           <height>25</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>25</height>
          </size>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
        </widget>
        <widget class="QTextEdit" name="bTextEdit">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>220</y>
           <width>91</width>
           <height>25</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>25</height>
          </size>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
        </widget>
        <widget class="QLabel" name="cLabel">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>250</y>
           <width>5</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>с</string>
         </property>
        </widget>
        <widget class="QLabel" name="dLabel">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>280</y>
           <width>6</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>d</string>
         </property>
        </widget>
        <widget class="QTextEdit" name="cTextEdit">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>250</y>
           <width>91</width>
           <height>25</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>25</height>
          </size>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
        </widget>
        <widget class="QTextEdit" name="dTextEdit">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>280</y>
           <width>91</width>
           <height>25</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>25</height>
          </size>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
        </widget>
        <widget class="QTextEdit" name="energyTextEdit">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>80</y>
           <width>91</width>
           <height>25</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>25</height>
          </size>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
        </widget>
        <widget class="QTextEdit" name="amountTextEdit">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>110</y>
           <width>91</width>
           <height>25</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>25</height>
          </size>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
        </widget>
        <widget class="QLabel" name="energyLabel">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>80</y>
           <width>131</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>Энергия электронов, эВ</string>
         </property>
        </widget>
        <widget class="QLabel" name="amountLabel">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>110</y>
           <width>104</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>Число электронов</string>
         </property>
        </widget>
        <widget class="QLabel" name="equationLabel">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>310</y>
           <width>461</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QComboBox" name="targetComboBox">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>140</y>
           <width>91</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QComboBox" name="spectrumComboBox">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>96</y>
           <width>271</width>
           <height>41</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>140</y>
           <width>111</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Материал мишени</string>
         </property>
        </widget>
        <widget class="QPushButton" name="calculatePushButton">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>330</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Рассчитать</string>
         </property>
        </widget>
        <widget class="QComboBox" name="methodComboBox">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>10</y>
           <width>331</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="answerLabel_1">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>350</y>
           <width>431</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="answerLabel_2">
         <property name="geometry">
          <rect>
           <x>4</x>
           <y>380</y>
           <width>431</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QCheckBox" name="filterCheckBox">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>160</y>
           <width>201</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Фильтр</string>
         </property>
        </widget>
        <widget class="QLabel" name="answerLabel_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>310</y>
           <width>431</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <widget class="QCheckBox" name="recalculateCheckBox">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>10</y>
           <width>351</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Пересчитать сечения</string>
         </property>
        </widget>
        <widget class="QLabel" name="minLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>171</width>
           <height>20</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Минимальная энергия, эВ</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QTextEdit" name="minTextEdit">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>40</y>
           <width>200</width>
           <height>25</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>25</height>
          </size>
         </property>
        </widget>
        <widget class="QLabel" name="maxLabel">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>70</y>
           <width>161</width>
           <height>20</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Максимальная энергия, эВ</string>
         </property>
        </widget>
        <widget class="QTextEdit" name="maxTextEdit">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>70</y>
           <width>200</width>
           <height>25</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>25</height>
          </size>
         </property>
        </widget>
        <widget class="QLabel" name="stepLabel">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>100</y>
           <width>151</width>
           <height>20</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Число шагов по энергии</string>
         </property>
        </widget>
        <widget class="QTextEdit" name="stepTextEdit">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>100</y>
           <width>200</width>
           <height>25</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>25</height>
          </size>
         </property>
        </widget>
        <widget class="QComboBox" name="ProgramChannelBox">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>10</y>
           <width>131</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="plotLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>360</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>946</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
